### DISCUSSION TITLE
Database is locked

### DISCUSSION THREAD

**Discussion by rjgrandy** [photoprism/photoprism#1358] [General]
I get a lot of errors like the following in my log file:

2021-06-05 09:47:42 ERRO database is locked

Are these of any concern? What do they mean?

**Comment by lastzero** [comment]
MariaDB, MySQL or SQLite?

**Reply by lastzero** [reply]
If SQLite, use MariaDB instead as SQLite uses file based locking. Not good when using mulitiple CPU cores.
